handbook/product/user_research/user_research_observer.md

```diff
@@ -0,0 +1,39 @@
+# So you’re about to help us with user testing
+
+Thank you so much for helping us with our user research! Observing and notetaking during user interviews and testing sessions is an amazing way for you to get a first-hand look at how others use our products and ideas, and your role as a notetaker is a huge help for the team.
+
+Here’s everything you need to know ahead of time.
+
+## Prerequisites
+
+- We use Lookback to conduct the sessions. Lookback requires Chrome, so please make sure you have Chrome installed and ready.
```

Comment:
```suggestion
- We use [Lookback](https://lookback.io/) to conduct the sessions. Lookback requires Chrome, so please make sure you have Chrome installed and ready.
```

Reply:
If observers have to log in, we should add the information here as well :) @quinnkeast